toolkit/scripts/imggen.mk

```diff
@@ -166,7 +171,7 @@ $(image_external_package_cache_summary): $(cached_file) $(go-imagepkgfetcher) $(
 # Stand alone target to build just the initrd, should not be used in conjunction with other targets. Use the 'iso' target instead.
 initrd: $(go-liveinstaller) $(go-imager)
```

Comment:
Add a dependency on `$(depends_CONFIG_INITRD)` here Make knows which components need to be rebuilt if the user changes the value of `CONFIG_INITRD`.

Reply:
thanks!  Actually the 2nd option is what I'm going for and Pawel also suggested "make image" on my earlier PR attempt.  Closing this PR now as "make image" seems to be doing the trick!